src/routes/transactions/__tests__/controllers/list-incoming-transfers-by-safe.transactions.controller.spec.ts

```diff
@@ -129,6 +144,21 @@ describe('List incoming transfers by Safe - Transactions Controller (Unit)', ()
     const chainId = faker.string.numeric();
     const safeAddress = faker.finance.ethereumAddress();
     const chain = chainBuilder().with('chainId', chainId).build();
+    const safe = safeBuilder()
+      .with('address', '0x4127839cdf4F73d9fC9a2C2861d8d1799e9DF40C')
```

Comment:
```suggestion
    const safeAddress = '0x4127839cdf4F73d9fC9a2C2861d8d1799e9DF40C';
    const chain = chainBuilder().with('chainId', chainId).build();
    const safe = safeBuilder()
      .with('address', safeAddress)
```

Reply:
Same as https://github.com/safe-global/safe-client-gateway-nest/pull/503#discussion_r1243921579

Reply:
Changed in [0450df9](https://github.com/safe-global/safe-client-gateway-nest/pull/503/commits/0450df9f6fa71b7b2f5bd8b364e0fa0aedbdb2f3)